app/code/Magento/StoreGraphQl/Controller/HttpHeaderProcessor/StoreProcessor.php

```diff
@@ -95,4 +110,19 @@ private function isAlreadySet(): bool
 
         return $this->httpContext->getValue($storeKey) !== null;
     }
-}
+
+    /**
+     * Check if provided store code exist
+     *
+     * @param string $storeCode
+     * @return bool
+     */
+    private function isStoreValid(string $storeCode): bool
+    {
+        $stores = $this->storeManager->getStores(true, true);
+        if (isset($stores[$storeCode])) {
+            return true;
+        }
+        return false;
+    }
+}
```

Comment:
Please add empty line at the end of file

Reply:
hi @Den4ik, sorry I didn’t get you. Where do you want me to add an empty line?

Reply:
![2021-09-20-15-34-38@2x](https://user-images.githubusercontent.com/1129623/134003114-9095a5ab-d830-44cf-95cf-781d0d3c038a.jpg)

Reply:
oh ok @Den4ik, I think that is not required. Can’t we ignore it?

Reply:
It will not pass static tests without empty line

Reply:
@Den4ik static test is fixed now, failing mftf test seems not related to this PR and I am not sure about Semantic Version Check